ee/codegen/src/generators/chat-message-content.ts

```diff
@@ -42,6 +43,8 @@ export class ChatMessageContent extends AstNode {
     } else if (this.chatMessageContent.type === "FUNCTION_CALL") {
       this.addReference(this.getFunctionCallChatMessageContentRef());
       this.addReference(this.getFunctionCallChatMessageContentValueRef());
+    } else if (this.chatMessageContent.type === "AUDIO") {
+      this.addReference(this.getAudioChatMessageContentRef());
```

Comment:
Same comment from this PR about using this.addReferences vs this.inheritReferences:
https://github.com/vellum-ai/vellum-python-sdks/pull/305